test/fit_tests/tests/rackhd20/test_rackhd20_api_rack_node.py

```diff
@@ -0,0 +1,160 @@
+'''
+Copyright 2016, EMC, Inc.
+
+Author(s):
+Norton Luo
+

```

Comment:
Can you please add some comments / description to this script?
What all is it testing.  thanks!